Makefile

```diff
@@ -0,0 +1,15 @@
+docker_image = resinos
+
+all: build_docker_image run_test
+
+build_docker_image: Dockerfile
+	@echo '[Info] Building docker image "$(docker_image)"...'
+	docker build -t $(docker_image) .
```

Comment:
Can we pass the pre-requisite directly instead of using a dot? i.e. `docker build -t $(docker_image) $<`